Update log-analytics-alerts.md

jboeshart opened the pull request:
Fixed a non-working query and updated the example for thresholds in metric measurements.

azurecla commented:
Hi __@jerriep__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

jboeshart committed:
Update log-analytics-alerts.md

Fixed a non-working query and updated the example for thresholds in metric measurements.
